app/src/main/java/io/github/droidkaigi/confsched2018/presentation/search/SearchFragment.kt

```diff
@@ -236,10 +236,21 @@ class SearchBeforeViewPagerAdapter(
         }
     }
 
-    enum class Tab(@StringRes val title: Int, val fragment: Fragment) {
-        Session(R.string.search_before_tab_session, SearchSessionsFragment.newInstance()),
-        Topic(R.string.search_before_tab_topic, SearchTopicsFragment.newInstance()),
-        Speakers(R.string.search_before_tab_speaker, SearchSpeakersFragment.newInstance());
+    enum class Tab(@StringRes val title: Int) {
+        Session(R.string.search_before_tab_session) {
```

Comment:
This is originally, if you have time, can you use capital letters? 🙇 
https://android.github.io/kotlin-guides/style.html#enum-classes

Reply:
Okay! Please wait a little 🙏